### PR TITLE
Add readability shortener

### LINKED ISSUE #2: Add Readability shortener
Readability shortener can work without API key what is great for quick start with short links. I'm ready to implement this feature so I'm just indicate my next step.
[API docs](https://www.readability.com/developers/api/shortener)

### PR DESCRIPTION
**Pull request by G07cha**
[Readability shortener](https://www.readability.com/developers/api/shortener) don't requires any tokens or API keys so it good for fast prototyping.
Also fixed indentation in `shortener_test.js`.

Resolves #2 
#7

### CODE REVIEW

File: test/shortener_test.js [hunk 1]
```diff
@@ -16,18 +16,31 @@ exports['urlshorener goo.gle'] = function (test) {
...
+					   }
+					   , "Deep equal for bit.ly");
     test.done();
 	});
 };
 
+exports['urlshorener readability'] = function (test) {
+  	var url = 'https://www.readability.com/';
+  	var s = new shortener.Shortener();
+  	s.bitly(url, function(result) {
```
Comment (khiro) [inline test/shortener_test.js]: I think s.bitly() is called in stead of s.readability().

**Comment by khiro** [comment]
Thanks @G07cha ! This PR is almost fine.
Please check commented line.

File: test/shortener_test.js (hunk 1 above)
Comment (G07cha) [inline test/shortener_test.js]: This is a shame for me. Fixed.
Also, can I know how do you run tests? Because I'm just checking with `sample.js` file.

**Comment by G07cha** [comment]
@khiro, fixed!

**Comment by khiro** [comment]
Thanks, test was successes.
I will merge this.

File: test/shortener_test.js (hunk 1 above)
Comment (khiro) [inline test/shortener_test.js]: np. I use "nodeunit" to run this test with a patch includes keys.

**Pull request merged**